dapps/marketplace/src/components/AboutParty.js

```diff
@@ -14,7 +14,35 @@ import Link from 'components/Link'
 
 import query from 'queries/Identity'
 import withOwner from 'hoc/withOwner'
-import { mapVerifiedAttestations } from 'utils/profileTools'
+
+const getVerifiedTooltip = provider => {
+  switch (provider) {
+    case 'email':
```

Comment:
Looks like the brands are translated in profileUtils... maybe we can use that as an fbt param to prevent duplication?

Reply:
It looks like you have removed those Tooltip components in your `nick/dapp3` branch. So we may not need this function at all.

Reply:
I kept them, just moved to Attestations component

Reply:
Got it. Will make this change